.github/workflows/build.yaml

```diff
@@ -70,13 +62,17 @@ jobs:
           if [ "${{ runner.os }}" = "Linux" ]; then
             sudo apt-get update
 
-            packages="elfutils libhiredis-dev libzstd-dev ninja-build pkg-config python3 redis-server redis-tools"
             # Install ld.gold (binutils) and ld.lld (lld) on different runs.
-            if [ "${{ matrix.config.os }}" = "ubuntu-18.04" ]; then
-              sudo apt-get install -y $packages binutils
-            else
-              sudo apt-get install -y $packages lld
-            fi
```

Comment:
binutils was selected for ubuntu-18.04 not because lld doesn't exist in Ubuntu 18.04 but to have a job that uses ld.gold instead of ld.lld (see the comment on line 74 above). I suggest keeping this test and making ubuntu-20.04 and ubuntu-22.04 run different linkers.

Reply:
Done